admin.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href=""/>
+    <link rel="stylesheet" href="app.css">
+    <title>Admin</title>
+    </head>
+    <body>
+        <!--Start First Section-->
```

Comment:
The code is small and it's clear. No no need to add comments because they will make the html code big later.